core/src/main/java/nl/nn/adapterframework/pipes/LogContextPipe.java

```diff
@@ -56,12 +57,15 @@ public PipeRunResult doPipe(Message message, PipeLineSession session) throws Pip
 		if (!getParameterList().isEmpty()) {
 			Map<String,String> values = new LinkedHashMap<>();
 			try {
+				if (!message.isRepeatable()) {
+					message.preserve();
+				}
 				ParameterValueList pvl = getParameterList().getValues(message, session);
 				for(ParameterValue pv : pvl) {
 					values.put(pv.getName(), pv.asStringValue());
 				}
-			} catch (ParameterException e) {
-				throw new PipeRunException(this, "exception extracting parameters", e);
+			} catch (Exception e) {
+				log.warn("Exception getting parameter values. Ignoring.", e);
```

Comment:
Just wondering if this is enough info for the end user (integration specialist) to understand/find the configuration which is wrong. @Laurens-makel what is your opinion?

Reply:
Depends on what is in `e`... Does it mention which parameter caused the exception?

Reply:
> Depends on what is in `e`... Does it mention which parameter caused the exception?
No, it only mentions the stacktrace, which isn't helping.